examples/dynamic-imports/static.config.js

```diff
@@ -36,4 +58,61 @@ export default {
       },
     ]
   },
+  renderToHtml: (renderToString, App, meta, prodStats) => {
+    const chunkNames = []
+    const appHtml = renderToString(
+      <ReportChunks report={chunkName => chunkNames.push(chunkName)}>
+        <App />
+      </ReportChunks>,
+    )
+
+    const { scripts } = flushChunks(prodStats, {
+      chunkNames,
+    })
+
+    meta.scripts = scripts.filter(script => script.split('.')[0] !== 'app')
+    return appHtml
+  },
+  Document: ({ Html, Head, Body, children, siteProps, renderMeta }) => (
+    <Html lang="en-US">
+      <Head>
+        <meta charSet="UTF-8" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0"
+        />
+      </Head>
+      <Body className="slug-home">
+        {children}
+        {renderMeta.scripts &&
```

Comment:
If we move all the other things to core... then to the user you just need to update renderToHtml and Document... which I feel is much less scary